src/main/java/com/PhamKornbluhGroup/PublicStashTabBulkAPICaller.java

```diff
@@ -0,0 +1,66 @@
+package com.PhamKornbluhGroup;
+
+import java.io.FileWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Arrays;
+
+public class PublicStashTabBulkAPICaller {
+
+    public APIResultData callAndPrintResults() {
+        //b GGG Get URL
+        HttpURLConnection connection = null;
+        APIResultData resultData = new APIResultData();
+        try {
+            URL url = new URL("https://api.pathofexile.com/public-stash-tabs/pc");
+            connection = (HttpURLConnection) url.openConnection();
+
+            //b Configure connection
+            connection.setRequestMethod("GET");
+
+            String[] gggBearerToken = SecretsHelper.getFormattedGGGBearerToken();
+            connection.setRequestProperty(gggBearerToken[0], gggBearerToken[1]);
+
+            String[] gggUserAgent = SecretsHelper.getFormattedGGGBearerTokenUserAgent();
+            connection.setRequestProperty(gggUserAgent[0], gggUserAgent[1]);
+
+            // TODO: Determine if the following 2 properties are needed
+            // connection.setRequestProperty("Content-Type","application/json");
+
+            //b Connect here. Disconnect happens in finally block
+            connection.connect();
+
+            resultData.initResultData(connection);
+        }
+        catch (Exception e) {
+            // TODO: Add Logging
+            System.out.println(e.getMessage() + Arrays.toString(e.getStackTrace()));
+        }
+        finally {
+            try {
+                connection.disconnect();
+            }
+            catch (Exception e) {
+                // TODO: Add Logging
+                System.out.println("Got exception when closing API connection. Error = " + e.getMessage());
+            }
+        }
+
+        //b writer auto-closes
+        this.writeResults(resultData);
+
+    return resultData;
```

Comment:
please indent one more level for consistency

Reply:
Fixed. Thanks!